### ISSUE TITLE
Full Resolution in Filename Parses as wrong episode

### ISSUE THREAD

**Issue by hbednar** [Sonarr/Sonarr#4716]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

For any show that has a resolution of 1920x9xx and is an mkv file type. For some reason all episodes after this in the series are displayed as that file. The only way to stop this is find another episode that is not of this resolution and delete the 1920x9xx episode.

### Expected Behavior

Only the correct episode should be detected regardless of the resolution.

### Steps To Reproduce

Download any episode that is 1920x9xx

### Environment

```markdown
- OS: Debian 11 and Debian sid
- Sonarr:3.0.6.1357
- Docker Install: no
- Using Reverse Proxy: no
- Browser: any
```

### What branch are you running?

Develop

### Anything else?

[Log File](https://gist.githubusercontent.com/hbednar/ad033c567730117adab65f92bf24e13d/raw/e1a7f64903b5bce39f4508c3ef4883cbc37cb221/gistfile1.txt)
Below are examples of two series this is happening with both have an episode file with a resolution of 1920x9xx and now Sonarr thinks the 1920x9xx accounts for every single episode.

![image](https://user-images.githubusercontent.com/71215137/138884927-f8d467e5-3622-416b-8d7c-b449697e539d.png)
![image](https://user-images.githubusercontent.com/71215137/138885132-5bcd9403-dc32-47af-9732-53f351ba6080.png)

![image](https://user-images.githubusercontent.com/71215137/138885229-72d935d7-ad8b-416f-8e90-cef124d5a237.png)

![image](https://user-images.githubusercontent.com/71215137/138885288-3717822d-95e3-45eb-be2f-0c6165f90682.png)
